update-main.go

```diff
@@ -203,19 +203,19 @@ func getReleaseUpdate(updateURL string, noError bool) updateMessage {
 	if err != nil && noError {
 		return updateMsg
 	}
-	fatalIf((err), "Fetching updates failed. Please try again.", nil)
+	fatalIf((err), "Fetching updates failed. Please try again.")
 
 	// Parse the date if its valid.
 	latest, err := parseReleaseData(string(updateBody))
 	if err != nil && noError {
 		return updateMsg
 	}
-	fatalIf(err, "Please report this issue at https://github.com/minio/minio/issues.", nil)
+	fatalIf(err, "Please report this issue at https://github.com/minio/minio/issues.")
 
 	// Verify if the date is not zero.
 	if latest.IsZero() && !noError {
 		fatalIf((errors.New("")),
-			"Unable to validate any update available at this time. Please open an issue at https://github.com/minio/minio/issues", nil)
+			"Unable to validate any update available at this time. Please open an issue at https://github.com/minio/minio/issues")
 	}
```

Comment:
Failed to retrieve update notice. Please try again later. Please report this issue at https://github.com/minio/minio/issues